src/lxcfs.c

```diff
@@ -1254,6 +1261,9 @@ int main(int argc, char *argv[])
 
 	new_argv = &argv[optind];
 	new_argc = argc - optind;
+	if (new_argc == 0) {
+		usage();
+	}
```

Comment:
Doesn't seem like that would be needed and I think the order might be better sm like:
```c
if (new_argc == 0) {
        lxcfs_error("Missing mountpoint");
	goto out;
}

if (new_argc > 1)
        usage();
```